src/core/gui/dialog/SettingsDialog.cpp

```diff
@@ -301,6 +304,8 @@ void SettingsDialog::customStylusIconTypeChanged() {
     gtk_widget_set_sensitive(get("highlightCursorGrid"), showCursorHighlightOptions);
 }
 
+void SettingsDialog::iconThemeChanged() {}
```

Comment:
Is this needed?

Reply:
Currently it is not needed. I will remove it, if I can't find a purpose.